projects/step-frontend/src/lib/modules/execution/components/schedule-overview/scheduler-page/scheduler-page.component.ts

```diff
@@ -50,7 +50,13 @@ export class SchedulerPageComponent implements OnInit {
 
   ngOnInit(): void {
     this._schedulerService.getExecutionTaskById(this._taskIdFn()).subscribe((task) => {
-      this.state.task.set(task);
+      console.log('task loaded', task);
+      if (task === undefined) {
+        // the task was not found
+        this.state.task.set(null);
+      } else {
+        this.state.task.set(task);
+      }
```

Comment:
I noticed that you introduced "null" as an additional value for the task and plan. Why don't we just use undefined? I assume you tried that but I would like to understand ;)